.azure/pipelines/ci.yml

```diff
@@ -19,6 +40,8 @@ jobs:
   parameters:
     # Ensures the alignment of branch name and deployment params
     buildArgs: '/warnaserror:BUILD1001'
+    ${{ if or(eq(variables['Build.Reason'], 'Schedule'), and(eq(variables['Build.Reason'], 'Manual'), eq(parameters.runCodeQL3000, 'true'))) }}:
```

Comment:
Narrow to run only in _internal_ (scheduled or manual w/ clicked checkbox) builds